builtin/providers/vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -799,10 +798,10 @@ func resourceVSphereVirtualMachineCreate(d *schema.ResourceData, meta interface{
 						return fmt.Errorf("Cannot specify name of a template")
 					}
 					vm.template = v
-					if hasBootableDisk {
+					if vm.hasBootableVmdk {
```

Comment:
should this check include checking for multiple non-template disks with bootable?